kube_dns/README.md

```diff
@@ -0,0 +1,60 @@
+# Kube-dns Integration
+
+## Overview
+
+Get metrics from kube-dns service in real time to:
+
+* Visualize and monitor dns metrics collected via Kubernetes' kube-dns addon
+  through Prometheus
+
+See https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns for
+more informations about kube-dns
+
+## Installation
+
+Install the `dd-check-kube_dns` package manually or with your favorite configuration manager
+
+## Configuration
+
+Edit the `kube_dns.yaml` file to point to your server and port, set the masters to monitor
+
+### Using with service discovery
```

Comment:
Oh that's a good point. What about adding an auto_conf.yaml as well? [Here](https://github.com/DataDog/integrations-core/blob/5.13.2/redisdb/auto_conf.yaml) is an example for redis. We can use `k8s-dns-sidecar-amd64` as the common denominator in `docker_images`.